fiat-api/src/main/java/com/netflix/spinnaker/fiat/shared/FiatPermissionEvaluator.java

```diff
@@ -206,6 +212,36 @@ public void invalidatePermission(String username) {
     permissionsCache.invalidate(username);
   }
 
+  private boolean canCreate(String username, String resourceType, String resourceName) {
+    boolean canCreate = false;
+
+    try {
+      Response response =
```

Comment:
we may want to think about including the ResourceGroup objects into the UserPermission response so that we can evaluate this without the remote call.

Given that `CREATE` is not likely on a high rps call path I'm okay with doing that later if it proves to be a problem though